### PR TITLE
[unrar/mdnsresponder] only windows

### PR DESCRIPTION
**Pull request by Adela0814**

### CODE REVIEW

File: ports/unrar/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "unrar",
   "version-string": "5.8.1",
```
Comment (Cheney-W) [inline ports/unrar/vcpkg.json:3]:
```suggestion
  "version": "5.8.1",
```

File: ports/mdnsresponder/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "mdnsresponder",
   "version": "878.270.2",
+  "port-version": 1,
   "description": "The mDNSResponder project is a component of Bonjour, Apple's ease-of-use IP networking initiative.",
   "homepage": "https://developer.apple.com/bonjour/",
-  "supports": "!arm"
+  "supports": "windows & !(uwp | arm | arm64)"
```
Comment (strega-nil-ms) [inline ports/mdnsresponder/vcpkg.json:7]:
```suggestion
  "supports": "windows & !uwp & !arm"
```

**Review by strega-nil-ms — COMMENTED** [review]
`arm` is the same as `arm32 | arm64`, so you don't need to do `arm | arm64`

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**